.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     timeout-minutes: 30
 
     env:
-      NX_CI_EXECUTION_ID: ${{ matrix.nx_ci_execution_env }}-${{ github.run_id }}
+      NX_CI_EXECUTION_ENV: ${{ matrix.nx_ci_execution_env }}
```

Comment:
I think you want the `ID` to be the same across each environment.

We can then use the `NX_CI_EXECUTION_ENV` variable to create a different env for each platform that get grouped under the same CIPE.